test/distributed/algorithms/quantization/test_quantization.py

```diff
@@ -0,0 +1,111 @@
+import torch
+import os
+import torch.cuda
+import torch.distributed as dist
+import torch.distributed.algorithms.quantization.quantization as quant
+from torch.distributed.algorithms.quantization.quantization import DQuantType
+from torch.testing._internal.common_distributed import (
+    requires_gloo,
+    skip_if_lt_x_gpu,
+    requires_nccl,
+)
+from torch.testing._internal.distributed.distributed_test import (
+    DistributedTest, TestDistBackend, BACKEND
+)
+from torch.testing._internal.common_utils import sandcastle_skip_if
+
+def _build_tensor(size, value=None, dtype=torch.float, device_id=None):
+    if value is None:
+        value = size
+    if device_id is None:
+        return torch.empty(size, size, size, dtype=dtype).fill_(value)
+    else:
+        return torch.empty(size, size, size, dtype=dtype).fill_(value).cuda(device_id)
+
+class DistQuantizationTests(TestDistBackend, DistributedTest._DistTestBase):
+    def setUp(self):
+        super().setUp()
+        self._fork_processes()
```

Comment:
nit: I believe we are trying to move towards spawn over fork (various testing and deadlock issues with fork):
```
self._spawn_processes()
```

should be all you need.

Reply:
Done! thanks 😊

Reply:
When I used `self._spawn_processes()` all of my local tests passed  but the CI tests started complaining. But when I switched back to fork they passed. If that's okay I will land this code with fork and then use a separate commit for spawn.